assets/queries/cloudFormation/iot_policy_allows_action/test/negative.yaml

```diff
@@ -0,0 +1,15 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: A sample template
+Resources:
+    IoTPolicy:
+      Type: AWS::IoT::Policy
+      Properties:
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action:
+            - iot:Connect
+            Resource:
+            - arn:aws:iot:us-east-1:123456789012:client/client1
+        PolicyName: PolicyName
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline